bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -439,6 +438,12 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
           v.edit(i)
       })
 
+    private val nonUpdatedInvestigatorName = when(p.investigators.pi.firstName === "Principal" && p.investigators.pi.lastName === "Investigator") {
```

Comment:
There is another check that will do that, though it will make a ToDo rather than an Error